File: data-structures-and-algorithms-ashrafi/python/src/recursive/problem_4/problem_4.py
def to_binary(a):
    if a // 2 == 0: 
        print(a, end="")
    else: 
        to_binary(a // 2)
        print(a % 2, end="")
        

def main():
    num = 13
    
    to_binary(num)
    
    
if __name__ == "__main__":
    main()

File: data-structures-and-algorithms-ashrafi/java/src/recursive/problem4/Problem4.java
public class Problem4 {
    
    public static void toBinary(int a){
        if (a / 2 == 0) {
            System.out.print(a);
        } else {
            toBinary(a / 2);
            System.out.print(a % 2);
        }
    }

    public static void main(String[] args) {
        int num = 93;

        toBinary(num);
    }
}
